source/audit-logs/activating-audit-logs.rst

```diff
@@ -0,0 +1,134 @@
+###############################
+Log creation and access
+###############################
+
+.. warning::
+
+    The following service is still in a technical preview and as such, will have
+    limitations on features and scope. Any feedback is appreciated and can be
+    provided through a support ticket. Thank you.
+
+***************************************
+Activating logs for your projects
+***************************************
+
+To get started using Audit logs, you will first need to activate the
+logging functionality for your projects.
+
+Prerequisites
+===============
+
+Before going further, you will need to ensure that you have :ref:`sourced an
+openRC file<source-rc-file>` for your command line so that you are able to
+perform the commands mentioned in the following example.
+
+Creating a container to house your logs
+=======================================
+
+As the functionality of the audit log service allows you to aggregate logging
+from multiple projects into one place, you will need to choose a location that
+can store the logs from all of your projects. If you only have one project,
+this step is still relevant as you will need a place for your logs to be
+directed to regardless.
+
+This means our first step is to create an object storage container that will
+hold all of our logs. The reason that we choose an object storage container
+over something like traditional block storage is that the object storage
+service is easier to communicate with via API and will scale in size to
+accommodate our logs over time.
+
+For this example, we will create a container named *logs-repo*
+
+.. code-block:: bash
+
+    $ openstack container create logs-repo
+
+.. note::
+
+    The actions on your project will be recorded instantly, however there is a
+    delay between the action being recorded and the log updating inside your
+    object storage container. This delay should not be significant, but it is
+    worth mentioning for any potential automation considerations.
+
+Gather credentials for log access
+=================================
+
+Now that we have our container, we move on to the next step which is gather
+credentials for your account in order to access the necessary log data.
+
+For this step, you will need to create a new user account for your project
+that only has the `Object Storage` role assigned. This is to restrict the
+permissions of said user to only the object storage service as the EC2
+credentials which are required for this step otherwise have a much larger reach
+into the project than is required to capture the logs; and would therefore be
+a security hazard.
+
+The simplest way to create this account is to send an invitation to an existing
+user account on your project with `+object-storage` appended to their email.
+
+For example:
+
+.. image:: user-invite-object-store.png
+
+This will send an email to your existing account, but logically, on the cloud,
```

Comment:
This is confusing to me. It does not send an email to your existing Openstack account.

It sends the invitation to the same email inbox you might have already used (provided the receiver email server supports the `+` extension - most do).